Servers/UI/OJS.Servers.Ui/ClientApp/src/layout/content/PageContent.tsx

```diff
@@ -70,9 +77,12 @@ const routes = [
 const PageContent = () => (
     <main className={styles.main}>
         <Routes>
-            {routes.map(({ path, Element }) => (
-                <Route key={path} path={path} element={<Element />} />
-            ))}
+            {routes.map(({ path, Element, title }) => {
+                const ElementWithWrappers = withTitle(Element, title);
```

Comment:
As you have another PR with page wrapper, i.e. https://github.com/SoftUni-Internal/OpenJudgeSystem/pull/571, maybe extracting a `render` function will be better, i.e.
```
...

const renderRoute = (({path, Element, title}) => {
    const WrappedElement = asPage(withTitle(Element, title));
    return (
        <Route key={path} path={path} element={<WrappedElement />}
    );
};

<Routes>
            {routes.map(renderRoute)}
</Routes>